core-bundle/src/Resources/contao/library/Contao/Image.php

```diff
@@ -227,7 +227,7 @@ public function getImportantPart()
 	/**
 	 * Set the target height
 	 *
-	 * @param integer $targetHeight The target height
+	 * @param integer|float $targetHeight The target height
```

Comment:
Why is this necessary?

Reply:
Because tests are inputting floats (and the method is casting the inputs to int). :see_no_evil:

Reply:
I think this is a similar topic that would be solved by mistrusting the annotated types.

Reply:
(This was also handled in daf0e0407d50b06b150e4997865c4629c4cdc71d.)